src/components/page/head.vue

```diff
@@ -39,7 +42,16 @@ defineOptions({
   margin-top: 0;
 }
 
-#page-head-title { overflow-wrap: break-word; }
+#page-head-title-infonav {
+  padding: 10px 0px;
+  display: flex;
+  align-items: center;
+}
+
+#page-head-title {
+  overflow-wrap: break-word;
+  overflow: hidden;
```

Comment:
@matthew-white I added this CSS to the page title for all things using page head. It seems to make a page title always wrap, even if it cant break on a word. 
<img width="1170" alt="Screenshot 2025-03-19 at 1 18 00 PM" src="https://github.com/user-attachments/assets/c224c249-09ed-4977-aa2e-ff5d9bb6f8e3" />